docs/events/gsoc/2024/gradle-build-server-android.md

```diff
@@ -107,23 +115,36 @@ As the popularity of Android development grows, the need for efficient build aut
 - My implementation takes into account any user defined build variants and the default build variants - debug and release except the default test variants - test and androidTest. ([Issue #183](https://github.com/microsoft/build-server-for-gradle/issues/183))
 - Android Components to be used by an Android project is configured via ANDROID_HOME environment variable. If the property doesn't exist then we are not providing the dependency. This implementation can be improved via fallback logic similar to what we did for JAVA_HOME in [PR #165](https://github.com/microsoft/build-server-for-gradle/pull/165). ([Issue #184](https://github.com/microsoft/build-server-for-gradle/issues/184))
 
-**Supporting Diagrams:**
+<details>
+
+<summary><b>Supporting Diagrams:</b></summary>
 
 ![Android Build Process](./images/AndroidImplementedBuildProcess.png)
 ![Android SourceSet Building](./images/SourceSetBuildingAndroid.png)
 
+</details>
+
 ## Documentation:
 
-- Detailed developer documentation for the implemented functionalities (composite builds, Java Home detection and sending notification on incompatibility, android support).
```

Comment:
Should it have a status line above, as other deliverables?

Reply:
Are these supposed to be deliverables? I am not sure I thought it was a separate section for additional information.

Reply:
For me Documentation should be a part of the delivered feature. But, otherwise, it is a separate deliverable and needs a status

Reply:
Addressed in https://github.com/gradle/community/pull/66/commits/a828ede6187833faa1f4c50e29428fea841298db